flight/views.py

```diff
@@ -1,3 +1,15 @@
-from django.shortcuts import render
+import json
+from django.views  import View
+from django.http   import JsonResponse
+from flight.models import Service
```

Comment:
```suggestion
import json

from django.views  import View
from django.http   import JsonResponse

from flight.models import Service
```
모듈 특징별로 간격 한 줄씩 넣어주세요

Reply:
수정하겠습니다!